.codecov.yml

```diff
@@ -44,6 +44,7 @@ ignore:
   - "bfintrinsics/**"
   - "bfm/ioctl/arch/**"
   - "bfvmm/include/test/**"
+  - "bfdriver/**"
```

Comment:
why was this added? I'd like to work at reducing these. Not adding to them.

Reply:
The driver tests arent being built right now because we dropped shared library support